agent/Gopkg.lock

```diff
@@ -172,8 +168,13 @@
 
 [[projects]]
   name = "github.com/fsouza/go-dockerclient"
-  packages = ["."]
-  revision = "8a48995398824064442c99cb45f453fb10c0154e"
+  packages = [
+    ".",
+    "internal/archive",
```

Comment:
Why are we moving away from `docker/docker/pkg/archive,jsonmessage,term`?

Reply:
I think these are the new dependency added by `go-dockerclient`, see this [commit](https://github.com/fsouza/go-dockerclient/commit/86502e0dad1897178b2bf308d6a5fd3a622cf727).